docs/src/content/installation/nix.md

```diff
@@ -1,4 +1,4 @@
-# Building from the source code using nix
+# Building the binary using nix
```

Comment:
I believe we should capitalize Nix as a proper name

Reply:
Done